### PR TITLE
fix: add frontend linter to CI

### PR DESCRIPTION
**Pull request by SunshineZerg**
Fix: https://github.com/casbin/confita/issues/16

### CODE REVIEW

File: web/src/CodeListPage.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ class CodeListPage extends React.Component {
         <Table columns={columns} dataSource={codes} rowKey="name" size="middle" bordered pagination={{pageSize: 100}}
           title={() => (
             <div>
-              {i18next.t("general:Code")}&nbsp;&nbsp;&nbsp;&nbsp;
+              {i18next.t("general:Codes")}&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (hsluoyz) [inline web/src/CodeListPage.js:142]: Code

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (hsluoyz) [inline yarn.lock:1]: Delete it

File: web/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,12539 @@
+{
```
Comment (hsluoyz) [inline web/package-lock.json:1]: Delete it

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by SunshineZerg** [comment]
Try adding new comment to refresh the pr

**Pull request closed without merging**